tests/bsim/bluetooth/audio/src/has_test.c

```diff
@@ -114,6 +121,7 @@ static void test_offline_behavior(void)
 	test_common();
 
 	WAIT_FOR_FLAG(flag_connected);
+	start_adv();
 	WAIT_FOR_UNSET_FLAG(flag_connected);
```

Comment:
Probably not incorrect, but would it have made more sense to wait for the disconnect before restarting advertisements?

Reply:
You are right. Thanks :)

I switched the order and pushed the change just now.